### ISSUE TITLE
How to adapt ESSNet for Object Detection, and train models on specifc classes with bboxes from COCO+LVIS+ADE20K?

### ISSUE THREAD

**Issue by Bukkster** [shipra25jain/ESSNet#3]
Hey there, 

I was wondering if it would be possible to adapt ESSNet to fine tune the encoder/decoder ouput tensor to create bounding boxes instead segmentation masks. If so any tips on how I can adapt the code and also your dataset py files to only import certain classes from the different datasets and generate labbeled bounding boxes from the ground truth segmentation masks and colour map?

I want to deploy the trained object detection model on a Pi4 (8gb Ram & 64bit RPi OS) for detecting smart home devices. Oddly enough there doesn't seem to be any labeled object detection datasets or pre-trained models on home appliances or things like windows, curtains, doors, closets, and other automateble objects. There are however parts of segmentation datasets that include these items but not enough of each class in any one dataset :/

Sorry if this is a dumb help request, I'm new to CV and DL.
